src/sys/socket/mod.rs

```diff
@@ -356,6 +366,9 @@ libc_bitflags! {
                   target_os = "fuchsia",
                   target_os = "freebsd"))]
         MSG_WAITFORONE;
+        /// Indicates that this message is not a user message but an SCTP notification.
+        #[cfg(linux)]
```

Comment:
If the alias only contains 1 target, we don't use it:

```suggestion
        #[cfg(target_os = "linux")]
```